setup.py

```diff
@@ -9,5 +9,6 @@
       install_requires=[
           'schematics==1.1.1',
           'requests==2.13.0'
```

Comment:
you need a comma on this line, and ideally you should add one on the next line as well